docs/validation/quarterly-report/2024-q2/index.md

```diff
@@ -0,0 +1,126 @@
+# 2024 Q2: DEA Quarterly Validation Report
+
+:::{contents} In this report
+:local:
+:backlinks: none
+:::
+
+## Executive Summary
+
+This Quarterly report summarises validation for DEA surface reflectance products for Quarter 2 (April-June) of 2024
+and presents aggregate validation results to the end of this quarter.
+
+* During this quarter, 2 field sites were measured 3 times and can be matched to 3 overpasses.
```

Comment:
Is it correct to say: 2 field sites were **each** measured 3 times, **to capture** 3 overpasses.

Reply:
During this quarter, a total of 3 measurements were taken across 2 sites, to capture 3 overpasses.